packages/nextjs/test/integration/.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true
+}
```

Comment:
Can we remove?

Reply:
Ah yeah. Next.js 13 seems to auto-generate this - I gitignored it.